sdk/servicebus/service-bus/src/core/streamingReceiver.ts

```diff
@@ -75,20 +76,34 @@ export class StreamingReceiver extends MessageReceiver {
    */
   static async create(
     context: ClientEntityContext,
-    options?: ReceiveOptions
+    options?: ReceiveOptions &
+      Pick<OperationOptions, "abortSignal"> & {
+        _createStreamingReceiver?: (
+          context: ClientEntityContext,
+          options?: ReceiveOptions
+        ) => StreamingReceiver;
+      }
   ): Promise<StreamingReceiver> {
     throwErrorIfConnectionClosed(context.namespace);
     if (!options) options = {};
     if (options.autoComplete == null) options.autoComplete = true;
-    const sReceiver = new StreamingReceiver(context, options);
+
+    let sReceiver: StreamingReceiver;
+
+    if (options?._createStreamingReceiver) {
+      sReceiver = options._createStreamingReceiver(context, options);
+    } else {
+      sReceiver = new StreamingReceiver(context, options);
+    }
 
     const config: RetryConfig<void> = {
       operation: () => {
-        return sReceiver._init();
+        return sReceiver._init(undefined, options?.abortSignal);
```

Comment:
Are we supposed to get the options from somewhere instead of "undefined"?
I can see you have retained the existing behavior, just asking if this should be something else!

Reply:
StreamingReceiver does not pass anything - internally there's a "get options" thing that basically just defaults to the right values for the initial set of handlers.

BatchingReceiver does pass an argument there, however, and it's a base class method so...I have to accommodate it.